fleet/api/http/openapi.yaml

```diff
@@ -250,6 +250,82 @@ paths:
           description: Missing or invalid content type.
         '500':
           $ref: "#/components/responses/ServiceErrorRes"
+  /agents/backends:
+    parameters:
+      - $ref: "#/components/parameters/Authorization"
+    get:
+      summary: 'List of Agent backends'
+      operationId: listAgentsBackends
+      tags:
+        - agents
+      responses:
+        '200':
+          $ref: "#/components/responses/AgentBackendsObjRes"
+        '400':
+          description: Failed due to malformed query parameters.
+        '401':
+          description: Missing or invalid access token provided.
+        '404':
+          description: A non-existent entity request.
+        '500':
+          $ref: "#/components/responses/ServiceErrorRes"
+  /agents/backends/pktvisor/taps:
+    parameters:
+      - $ref: "#/components/parameters/Authorization"
+    get:
+      summary: 'List of Taps available from current agents'
```

Comment:
let's make this description very pktvisor specific: "List of pktvisor Taps available from current agents"